lectures/12-plonk.tex

```diff
@@ -0,0 +1,313 @@
+\documentclass[../lecture-notes.tex]{subfiles}
+\usepackage[T1]{fontenc}
+\usepackage{lmodern}
+\usepackage{xcolor}
+\usepackage{fix-cm}
+\usepackage{amssymb}
+\pgfplotsset{compat=1.18}
+
+\begin{document}
+\subsection{Preliminaries}
+Before delving into Plonk, let's first consider a few useful proving system gadgets.
+
+Let $\Omega$ be some multiplicative cyclic subgroup (recall the \ref{section:mult-group-finite-field}) of $\mathbb{F}_p$ of size $k$. 
+
+Let $f \in \mathbb{F}_p^{(\leq d)}[X] \qquad (d \geq k)$ \begin{tabular}{|c|}\hline Verifier has $f$ \\ \hline \end{tabular} 
+
+Now we need to construct efficient Poly-IOPs for two tasks described in below sections.
```

Comment:
Also, I believe you omitted the product check on $\Omega$ since it is not used for classical Plonkish arithmetization (with no custom gates), but I suggest adding it nonetheless